scripts/integration-tests/run-curiosity.sh

```diff
@@ -0,0 +1,21 @@
+if [[ -z $STATIC_PATH ]]; then
+    scriptdir=$(realpath -s "$0" | xargs dirname)
+    echo "Missing static files."
+    echo "Did you read $scriptdir/readme.md ?"
+    exit 1
+fi
+
+echo "[+] Setting up the curiosity runtime env"
+rundir=$(mktemp -d)
+trap 'rm -rf -- "$rundir"' EXIT
+cd "$rundir"
+
+# Init curiosity env
+cty init
+cp -r "$STATIC_PATH"/* "$rundir"
```

Comment:
I would think it is not necessary to copy that directory: you can refer to its path in the Nix store directly.

Alternatively, you can set and export the existing `CURIOSITY_STATIC_DIR` environment variable, which is read by `cty serve` (i.e. no need for the `--static-dir` option when that environment variable is set).

B.t.w. this is exactly what the `nix-shell default -A shell` shell does, so I think it makes sense that your `run-interactive-integration-env` build artefact would be part of that shell.